Dockerfile

```diff
@@ -56,6 +56,17 @@ ADD ./get /home/ipython/py/get
 ENV PYTHONPATH /home/ipython/py/:$PYTHONPATH
 ENV PATH /home/ipython/py/:$PATH
 
+# ENV variables to replace conf file
+ENV DEBUG false
```

Comment:
Just in one ENV command. Dockerfiles are restricted to X commands. We should be spare.